.github/workflows/scripts/monitor-typescript.ts

```diff
@@ -53,6 +67,19 @@ function parseContentsToKeyValue(
           acc: Record<string, {unit: string | undefined; value: number}>,
           line: string
         ) => {
+          const match = line.match(/Done in ([0-9]+\.[0-9]+)s/);
+          if (match) {
+            // The total time from the diagnostics does not include the
+            // time spent generating the trace, so we take the duration
+            // of the yarn command.
+            const parsedValue = parseFloat(match[1]);
+            acc['User Time'] = {
```

Comment:
Is this not more like "real time"? iirc user time means cumulative between all CPUs